src/components/ErrorPage.tsx

```diff
@@ -8,9 +8,32 @@ import {
   faTrash,
 } from '@fortawesome/free-solid-svg-icons'
 
+/** Type narrowing function of unknown error to a string */
+function errorMessage(error: unknown): string {
+  if (isRouteErrorResponse(error)) {
+    return `${error.status} ${error.statusText}`
+  } else if (error != undefined && error instanceof Error) {
+    return error.message
+  } else if (error instanceof Object) {
```

Comment:
I would generally avoid using `instanceof` checks for built-ins. Because of [frames](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/instanceof#instanceof_and_multiple_realms), they don't always do what you expect.

I might do something like this:

```suggestion
  } else if (error && typeof error === 'object') {
```